include/CL/opencl.hpp

```diff
@@ -6730,7 +6731,7 @@ class Program : public detail::Wrapper<cl_program>
             notifyFptr,
             data);
 
-        BuildLogType buildLog(1);
+        BuildLogType buildLog(0);
```

Comment:
I will open an issue on CLHPP about this, because an innocent getInfo/getBuildLog on default constructed Device and Program objects should not cause crashes, but CL_INVALID_OBJECT. The fact that the wrapper returned these objects just made the issue inevitable.